lib/index.js

```diff
@@ -30,6 +30,7 @@ ytdl.getInfo = getInfo;
 ytdl.chooseFormat = util.chooseFormat;
 ytdl.filterFormats = util.filterFormats;
 ytdl.cache = cache;
+ytdl.validateId = util.validateLink;
```

Comment:
Would prefer if this is one line above so it can be grouped with the other functions from `util`